### PR TITLE
Add support to RKE v1.3.3

### PR DESCRIPTION
**Pull request by rawmind0**
Issues #308 and #309 

Requires https://github.com/rancher/rke/pull/2701

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -6,18 +6,15 @@ require (
...
-	gopkg.in/yaml.v2 v2.2.8
-	k8s.io/api v0.20.0
-	k8s.io/apimachinery v0.20.0
-	k8s.io/apiserver v0.20.0
-	k8s.io/client-go v12.0.0+incompatible
+	gopkg.in/yaml.v2 v2.4.0
+	k8s.io/api v0.21.0
+	k8s.io/apimachinery v0.21.0
+	k8s.io/apiserver v0.21.0
+	k8s.io/client-go v0.21.0
```
Comment (nickgerace) [inline go.mod:17]: Should we be using a new patch release?

Comment (rawmind0) [reply go.mod:17]: Using same go modules versions as [rke v1.3](https://github.com/rancher/rke/blob/release/v1.3/go.mod#L47)

**Comment by rawmind0** [comment]
CI is failed due to acceptances test are failing checking `secrets_encryption_config.custom_config` values. This is due to a bug at rke, that should be solved with PR https://github.com/rancher/rke/pull/2701

**Comment by stefandanaita** [comment]
Hi @rawmind0! Is this getting tagged soon? I'm after Apple M1 support 😢 Thank you!!

**Comment by rawmind0** [comment]
> Hi @rawmind0! Is this getting tagged soon? I'm after Apple M1 support 😢 Thank you!!

Hi @stefandanaita , this PR is blocked until rke PR https://github.com/rancher/rke/pull/2701 is merged. Anyway, just released v1.2.4 that is already providing darwin arm64 binaries

**Comment by hutm** [comment]
https://github.com/rancher/rke/pull/2701 seems to be merged. Are there any other blockers for this one?

**Comment by pwurbs** [comment]
Could someone please resolve the branch conflicts and merge. I want to deploy Rancher 2.6, but it requires a newer than the currently included RKE version.

**Comment by rawmind0** [comment]
PR updated to support rke 1.3.3-rc4 (with required patch applied). Once rke 1.3.3 is released this PR should be updated to use it.

**Comment by git-ival** [comment]
Would it be possible to bump the rke version to 1.3.3-rc5? Currently trying to make use of this extremely helpful module with the latest Rancher rc (2.6.3-rc4)!

**Comment by hutm** [comment]
@rawmind0 who would be the reviewer to ping?

**Comment by pwurbs** [comment]
> PR updated to support rke 1.3.3-rc4 (with required patch applied). Once rke 1.3.3 is released this PR should be updated to use it.

RKE 1.3.3 has been released 2 days go.

**Review by superseb — APPROVED** [review]

**Pull request merged**